### ISSUE TITLE
Newest Version gtp5g is not Respone the UPF GTP Reply

### ISSUE THREAD

**Issue by j13tw** [PrinzOwO/gtp5g#18]
Hello, PrinzOwO
I am testing the gtp5g in my free5gc v3.0.4 env
and I try to use the newest version to build gtp5g to my environment
the newest is the ```v1.0.0-f```
and I test in free5gc v3.0.4 to ping UPF IP,
It can unpack the GTP<ICMP_Request> from UE/RAN request
and get the <ICMP_Request> => <ICMP_Reply>
but not reply the GTP<ICMP_Reply> Packet on UPF requst to UE/RAN
and I try the old version in my forked gtp5g repo
It can reply GTP<ICMP_Reply> to UE

Old version
<img width="672" alt="截圖 2021-02-08 下午11 12 21" src="https://user-images.githubusercontent.com/27864573/107240457-e4a19280-6a64-11eb-95ec-635722936087.png">
Newest version
![截圖 2021-02-08 下午11 12 47](https://user-images.githubusercontent.com/27864573/107240465-e703ec80-6a64-11eb-930f-5f8202ab19f6.png)

Maybe you can tell me, gtp5g version v1.0.0-f 
It can mapping to free5gc which version when I testing.
thx!

**Comment by stevenchiu30801** [comment]
Same issue here. It seems that there are some bugs in the merged PR [d1ed954](https://github.com/PrinzOwO/gtp5g/commit/d1ed954696906dcfe1345282707af853df2e8336). I could make it work after checking out to previous commit.

Here are more details of the issues. There are TX errors on `upfgtp` tun interface with the decapsulated packets.
```
# ip -s -s -d link show upfgtp
5: upfgtp: <POINTOPOINT,MULTICAST,NOARP,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN mode DEFAULT group default qlen 1000
    link/none  promiscuity 0
    gtp5g addrgenmode eui64 numtxqueues 1 numrxqueues 1 gso_max_size 65536 gso_max_segs 65535
    RX: bytes  packets  errors  dropped overrun mcast
    252        3        0       0       0       0
    RX errors: length   crc     frame   fifo    missed
               0        0       0       0       0
    TX: bytes  packets  errors  dropped carrier collsns
    0          0        3       0       3       0
    TX errors: aborted  fifo   window heartbeat transns
               0        0       0       0       0
```

**Comment by stevenchiu30801** [comment]
Update: Fixed in the newest PR [919b6a3](https://github.com/PrinzOwO/gtp5g/commit/919b6a37c3c05bcfc7c52d25c08aa5f42f79979e).

**Comment by muthuramanecs03g** [comment]
@j13tw ,

This problem was got resolved by the latest code and closing it.
